uint64 Error_Type_Number
uint64 Power_Number
uint64 Module_Number
uint64 Block_Number
uint64 StartTime
uint64 EndTime
